sdlib/d/gc/page.d

```diff
@@ -725,6 +729,42 @@ private:
 					auto oldOccupancy = e.slabData.rawContent[i];
 					auto newOccupancy = oldOccupancy & bmp[i];
 
+					if (ec.supportsMetadata) {
```

Comment:
If you put that block at the end of the loop, you can early continue rather than nest like migrating birds.

Reply:
I need to do this before setting the bits in the slab, right? I have rewritten it, but I have to copy that line.